CHANGES.rst

```diff
@@ -72,6 +72,11 @@ Unreleased
 
 * :class:`.APIException` is deprecated and slotted for removal in PRAW 8.0.
 
+**Fixed**
+
+* :meth:`.SubredditFlair.update` will not error out when the flair text 
+  contains quotations.
```

Comment:
Not "quotations," please — "quote marks" (the quotation is the text contained within the quote marks).

Reply:
I don't know how to do the "suggest change directly" thing, or else I would.

Reply:
It's the same as making a code block and adding a language, but in this case, the language is suggestion:

```
\`\`\`suggestion
blah blah blah
\`\`\`
```

You can also click on the icon two icons to the left of the bold icon on the Github Web Editor, or do Cmd+G.

```suggestion
  contains quote marks.
```